db/migrate/20190811211131_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
```

Comment:
Todo: remove this migration after merge with @sametsJR

Reply:
ok